### DISCUSSION TITLE
[NEWS] Call for Testimonials!

### DISCUSSION THREAD

**Discussion by tabacitu** [Laravel-Backpack/community-forum#306] [Announcements]
We're _finally_ going to put a few testimonials on our homepage. 

Despite the fact that we've been around for 3+ years and thousands of developers use Backpack every month, we have NOT collected testimonials, ever 😞 Nor do we have a good way to reach out and ask for testimonials - I don't want to spam people, I would hate to receive such an email. We'll fix this for future users, but I don't want to wait a few more months.

Right now our list has a grand total of 1 testimonial. Shameful, I know 😄 

**If you would like to help us out with a few kind words, please PLEASE reply here** 😄 If you're concerned about your name/company/etc showing up, don't worry, it can be as little as:

> It made my development life _much_ easier. Get Backpack. You won't regret it. -- Todd, USA

Thanks a lot!

**Comment by lotarbo** [comment]
I am sorry, maybe I made mistakes in the sentence, coz i using google translate)

> The most customisable and functional Admin Panel for Laravel, out of all I've tried in my projects. Thank you! -- Bogdan, Russia

**Comment by tabacitu** [comment]
Thank you so much @lotarbo !

**Comment by azaricstefan** [comment]
The best admin panel for Laravel that exists! Using it since July 2018 and never had any serious issues. Backpack team is responsible, listens to the community and works hard on supporting new features. Running this admin panel in production for 2 years and I totally recommend it to everyone!
Stefan, Serbia
[Zakaži Tehnički](https://zakazitehnicki.rs)

**Comment by juventus18** [comment]
> Backpack speeds up CRUD development time and allows adding complex features (such as revision history) with just a few lines of code. Overriding default Backpack behavior (if you need to) is easy and well documented.  There is also a substantial community around Backpack that is always willing to help if you get stuck.
- Sean, USA

**Comment by AbbyJanke** [comment]
Best Administration Panel for Laravel! Not only has it helped me become a better developer. The community is so friendly and helpful to each other. I've made great personal projects using it and adapted it to be used on a project for my work and create applications used by several US States.

**Comment by Cadienvan** [comment]
Surely the starting point for every Laravel project. Don't reinvent the wheel, use the best wheel ever made!!

Michael, Italy

**Comment by rajendrap123** [comment]
_Amazing Laravel Package for every admin project needs. With lots of options and tools, it accelerates development of project. Kudos to the Team._

Rajendra, Nepal

**Comment by bedoz** [comment]
I have been using Backpack for 2 years, I tried it for a project and I immediately fell in love with it. The huge work they have done and that they are continuing to do in these years make me proud to be one  of the users for a long time. I also tried to merge some changes, but without success, anyway I think it's one of the best if not the best admin panel and the most flexible that I know.

Alex Bedendo, Italy

**Comment by iMokhles** [comment]
**Backpack for Laravel** is the most powerful and customizable admin panel for **Laravel** i'm using it since 2017 and i didn't regret one moment since i used it, it's easy to use and really has a great team who works hard, using it for almost 3 years made me understand other concepts to build CRUD and Admin Panels for **Laravel**, you have to try it and i'm sure you won't regret it at all.
Thanks **Backpack for Laravel** team and community to be part of our life

**Mokhlas Hussein, France**

**Comment by tabacitu** [comment]
Guys & gals, can I just say... I've ran through this thread again and it makes me _super-happy_ to know you think so highly of Backpack, and that you're happy with it. You've put a smile on my face - once again today. Thanks a lot, you're the best 🥳🙏

**Comment by christmex** [comment]
Am I late? sorry, but I really want to do something for this great tool

>Backpack for laravel Speed up the development process, and make the pain of development lower, I really love what Backpack for Laravel has done, thanks for creating this tool for us.

**Reply by o15a3d4l11s2** [reply]
It's never late, I am sure the devs will always be happy to hear their work and products are appreciated 😊

**Comment by munjaldevelopment** [comment]
"I have developed more than **30 projects using Laravel Backpack**. Recently developed a project using **Laravel Backpack along with Vue3 & I am getting great appreciation from client**. It's all happen due to @tabacitu & team Backpack. Looking forward for new updates & _really excited for Backpack V6_"
